.changeset/tiny-foxes-provide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-adr-backend': minor
```

Comment:
The backend changes should be non-breaking and this is pre-major v1 so patch is fine here
```suggestion
'@backstage/plugin-adr-backend': patch
```

Reply:
changed to patch :)